Makefile

```diff
@@ -208,6 +214,12 @@ release-verify: release-prereqs
 	# Run FV tests against the produced image.
 	$(MAKE) CONTAINER_NAME=calico/kube-controllers:$(VERSION) st
 
+## Generates release notes based on commits in this version.
+release-notes: release-prereqs
+	mkdir -p dist
```

Comment:
I'm undecided about using dist for this. Maybe release-{VERSION} would be better

Reply:
yes me too, I actually meant to change this but forgot.